client/lib/analytics/index.js

```diff
@@ -437,6 +462,7 @@ const analytics = {
 				}
 
 				window.ga( 'create', config( 'google_analytics_key' ), 'auto', parameters );
+				window.ga( 'set', 'anonymizeIp', true );
```

Comment:
Are we sure we want to bundle this with the banner? I feel like that might be something that could have weird consequence that could end up being rolled-back?

Reply:
That wasn't my impression, but if you have reservations I'm happy to move it to its own PR. And it does indeed make sense, it is a different subject.